### ISSUE TITLE
Fatal error in R (version 4.2.0)

### ISSUE THREAD

**Issue by samuelkim1011** [rspatial/terra#677]
I have been encountering fatal error when I tried to load terra package (version 1.5.34). The package worked just fine before the update, and all the other packages that I've been using (tidyverse, texreg, etc.) work perfectly well, so I'm guessing this has to do with some compatibility issues between terra and R.

Any help would be much appreciated!

P.S. I do not think any screenshot would be necessary, as "fatal error" pops up as soon as I type in library(terra).

**Comment by rhijmans** [comment]
This is probably because of a faulty install. Most likely because you are using R-studio and loading an old session before installing terra. This may help (and is best done in Rgui or the console)

If you are using Rstudio: close all tabs (scripts etc.) and then run

```
unlink("RData")
``` 
Quit R (R-Studio). Do *not* save the workspace
Restart and assure that 

`ls()` shows nothing and `sessionInfo()` only a few base packages. If not, repeat the above. 

Now run `install.packages("terra")`

**Comment by samuelkim1011** [comment]
Thanks rhijmans for your help! The thing is, I've already tried this before and it hasn't worked. Though I've given it a try again just to make sure - to no avail.

I tried removing and reinstalling both R GUI and Rstudio. Neither of these worked. Any other suggestions would be welcome!

**Comment by katherinerosewolf** [comment]
Having the same problem. The solution also didn't work for me.

**Comment by rhijmans** [comment]
It would be useful to know more. Can you show your entire session? That is start up R and run: 

```
sessionInfo()
install.packages("terra`)
```

This is what I get:
```

#Type 'demo()' for some demos, 'help()' for on-line help, or
#'help.start()' for an HTML browser interface to help.
#Type 'q()' to quit R.

sessionInfo()
#R Under development (unstable) (2022-05-14 r82360 ucrt)
#Platform: x86_64-w64-mingw32/x64 (64-bit)
#Running under: Windows 10 x64 (build 19043)
#
#Matrix products: default
#
#locale:
#[1] LC_COLLATE=English_United States.utf8 
#[2] LC_CTYPE=English_United States.utf8   
#[3] LC_MONETARY=English_United States.utf8
#[4] LC_NUMERIC=C                          
#[5] LC_TIME=English_United States.utf8    
#
#attached base packages:
#[1] stats     graphics  grDevices utils     datasets  methods   base     
#
#loaded via a namespace (and not attached):
#[1] compiler_4.3.0

install.packages("terra")
#--- Please select a CRAN mirror for use in this session ---
#trying URL 'https://cloud.r-project.org/bin/windows/contrib/4.3/terra_1.5-34.zip'
#Content type 'application/zip' length 24854755 bytes (23.7 MB)
#downloaded 23.7 MB
#
#package ‘terra’ successfully unpacked and MD5 sums checked
#
#The downloaded binary packages are in
#        C:\temp\RtmpOcl4fi\downloaded_packages
```

It may also help to first re-install the `Rcpp` package.

**Comment by katherinerosewolf** [comment]
I fixed it on my machine! I don't remember the exact steps I used, though, so I suspect that my `sessionInfo` might not be helpful anymore, alas. I think that what I did was delete the `terra` folder in my R 4.2 packages folder, close RStudio, restart RStudio, and then reinstall `terra` via RStudio's point-and-click menus ("Tools > Install Packages... > `terra` > Install") with R 4.2.0 running. I suspect that something referencing the package broke when I installed R 4.2.0 for the first time that reinstalling it fixed, but I have no idea what it was.
